less/mixins.less

```
@@ -1 +1,29 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+
+.sign-up-btn() {
+    width: 30%;
+    background-color: @btn-bg-color;
```

```suggestion
    background-color: @btn-bg-color;
```

Great use in leveraging the powers of Less